c7n/resources/account.py

```diff
@@ -96,12 +97,16 @@ class CloudTrailEnabled(Filter):
     permissions = ('cloudtrail:DescribeTrails', 'cloudtrail:GetTrailStatus')
 
     def process(self, resources, event=None):
-        client = local_session(
-            self.manager.session_factory).client('cloudtrail')
+        session = local_session(self.manager.session_factory)
+        client = session.client('cloudtrail')
         trails = client.describe_trails()['trailList']
         resources[0]['c7n:cloudtrails'] = trails
         if self.data.get('global-events'):
             trails = [t for t in trails if t.get('IncludeGlobalServiceEvents')]
+        if self.data.get('current-region'):
```

Comment:
this could use some docs in the class docstring. i do think we have other ways of getting/verifying this with the extant, but its a net win for policy authoring.